frontend/app/src/locales/en.json

```diff
@@ -2536,37 +2536,37 @@
     "binance": {
       "line_one": "Imports: Deposits, Withdrawals, Trades, ETH Staking, Small assets exchange BNB, ETH Staking Rewards, Launchpool Interest, POS savings interest, POS savings purchase, POS savings redemption.",
       "line_two": "Because of binance's csv format there is a chance that not all data will be imported. If this will happen, you will get warning.",
-      "name": "Binance",
-      "note": "Important notes for importing data from {0}'s CSV exports.",
-      "source_name": "Binance"
+      "name": "Binance"
     },
     "bisq": {
-      "import": "Import {0} from bisq",
+      "import": "Import {0} from Bisq",
       "import_trade": "trades",
       "line_one": "[Experimental] If you find an issue with the imported data, let us know so we can adapt the code accordingly.",
-      "name": "Bisq",
-      "note": "Important notes for importing data from {0}'s CSV exports.",
-      "source_name": "Bisq"
+      "name": "Bisq"
+    },
+    "bitcoin_tax": {
+      "import": "Import {0} from BitcoinTaxes",
```

Comment:
@dimyG is this text correct here? is this how it should appear?

Reply:
It should be [bitcoin.tax](https://bitcoin.tax/)